gym/tf_od_api/workspace/train_core/train.sh.old

```diff
@@ -0,0 +1,34 @@
+cd /opt/tf_od_api/workspace/train_core
```

Comment:
For consistency, we probably want the `$(pwd)` here as well (before the `tf_od_api/`). And I maintain the same suggestion here to define a global variable for the models location.

Reply:
there are paths that a meant to be relative, but about the model locations it is not a problem they are all stored on gym/ai4coral